query-node/mappings/src/membership.ts

```diff
@@ -326,6 +328,11 @@ export async function members_MemberProfileUpdated({ store, event }: EventContex
     }
   }
 
+  if (typeof metadata?.validatorAccount === 'string') {
+    member.metadata.validatorAccount = (metadata.validatorAccount || null) as string
```

Comment:
This could be:
```suggestion
  if (typeof metadata?.validatorAccount === 'string' && metadata.validatorAccount !== member.metadata.validatorAccount) {
    member.metadata.validatorAccount = metadata.validatorAccount || undefined
```
To prevent resetting the verification by mistake when the same account is passed and to allow removing the validator account by passing an empty string.

Reply:
removing `null` creates error during membership update test where `validatorAccount` should be set to null

Reply:
Good job I'm not sure but maybe `null` actually unset the value on hydra while `undefined` is ignored (I think I read an issue about something like that a while ago).